fish-rust/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
```

Comment:
You may want to use a [cargo workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html). This allows sharing the `Cargo.lock` and target dirs between all crates in the cargo workspace. You can do this by adding the following to this `Cargo.toml`:

```toml
[workspace]
members = ["path/to/crate1", "path/to/crate2"]
```

You can also use `*` as wildcard. And in this specific case I think you can omit the `members` key entirely as `widestring-suffix` is used as dependency of this crate.